src/queries/waterTemperature.js

```diff
@@ -0,0 +1,12 @@
+import { waterTemperatureEndpoint } from '../config';
+
+export const getWaterTemperature = async () => {
+  const fetchObj = {
+    method: 'GET',
+    headers: { Authorization: 'readonly:LjjhVGU46EBcKND9j3cX' }
```

Comment:
what is this auth? is this publicly allowed? this should go to the secrets or am i wrong?

Reply:
Yeah, you're right, I'll update. It's my fault.